app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -17,5 +17,4 @@
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
     <background android:drawable="@drawable/ic_launcher_background"/>
     <foreground android:drawable="@drawable/ic_launcher_foreground"/>
-    <monochrome android:drawable="@drawable/ic_launcher_foreground" />
```

Comment:
It looks like this is going to remove adaptive icon support, these files should probably remain unchanged?

Image Asset Studio through AS should probably have an option/add this in automatically too.